### PR TITLE
runtests: add result summary of execution cases

### PR DESCRIPTION
**Pull request by xupengfe**

### CODE REVIEW

File: runtests [hunk 1]
```diff
@@ -81,6 +85,9 @@ runtest() {
   else
     echo -e "<<<test end, result: $result, duration: ${duration}s>>\n"
   fi
+  case_result=""
```
Comment (hanning0511) [inline runtests:88]: better to put the definition of the variable at the top of the function.

Comment (xupengfe) [reply runtests:88]: It's already defined on top of the function:  "local case_result="""
Here  case_result=""  is cleaning the left result before get new case_result for  loop  usage,  and there is loop here.
So this line could be removed.
Thanks!

File: runtests [hunk 2]
```diff
@@ -138,6 +145,15 @@ if [[ -z "$CMDFILES" ]] && [[ -z "$CMDLINE" ]]; then
   err "no test to run!"
 fi
 
+START_TIME="$(date +%Y-%m-%d_%H-%M-%S)"
+SUMMRY_LOG="/tmp/lkvs_${START_TIME}_summary.log"
+echo "Test Start Time: $START_TIME" > "$SUMMRY_LOG"
+{
+  echo "--------------------------------------------------------"
+  echo "Testcase                                                                    Result     Exit Value  DURATION"
```
Comment (hanning0511) [inline runtests:153]: Better to use `printf` here to make this line short

```shell
printf "%-76s%-11s%-12s%-8s" "Testcase" "Result" "Return Code" "Duration"
```

File: runtests (hunk 1 above)
Comment (hanning0511) [reply runtests:88]: Got it. Thanks for the explanation.

File: runtests (hunk 2 above)
Comment (xupengfe) [reply runtests:153]: Thanks for suggestion!
I will improve it.
Thanks!

**Pull request closed without merging**